docs/docs/account/user-permissions.mdx

```diff
@@ -52,6 +56,12 @@ For organizations without a Pro or Enterprise account, user permissions are eval
 
 :::note
 
-If your organization has enabled the setting to allow verified users to automatically join your organization, they will receive the `collaborator` role by default when they join.
+If your organization has enabled the setting to allow verified users to automatically join your organization, they will receive the `Collaborator` role by default when they join. However, you can change your organizations' default role at the bottom of the Team page.
 
 :::
+
+### Teams
+
+Enterprise organizations using GrowthBook can create Teams with distinct capabilities. When setting up a Team, you have the option to define both a global role and project-level roles, much like how you do for individual users. Once a Team is established, multiple users can be added to it. Any user added to a Team will automatically inherit all permissions assigned to that Team. This feature becomes particularly useful when combined with [GrowthBook's SCIM integration](https://docs.growthbook.io/integrations/scim), enabling automated user provisioning and de-provisioning.
```

Comment:
On the SCIM page it says this:

> Do you support groups?
> Currently, we do not support groups, but we are actively working on adding group support in the future.

In general, I think we also need to update the SCIM docs page to reference groups

Reply:
I tackled that here: https://github.com/growthbook/growthbook/pull/1887

Reply:
sounds good.  Lets merge this PR now then and update and merge #1887 after